packages/electron-updater/src/AppUpdater.ts

```diff
@@ -51,6 +51,17 @@ export abstract class AppUpdater extends EventEmitter {
    */
   allowDowngrade = false
 
+  /**
+   * Web installer files might not have signature verification, this switch prevents to load them unless it is needed.
+   *
+   * Currently false to prevent breaking the current API, but it should be changed to default true at some point that
+   * breaking changes are allowed.
+   *
+   * @default false
+   */
+
+  disableWebInstaller = false
```

Comment:
Sounds good, we can leave it as false. I think we should log a warning then for the user that they should set the flag true if they're not using web installers, along with a note that it will default true in a future version.